bom/compile/pom.xml

```
@@ -18,6 +18,7 @@
 
   <properties>
     <californium.version>2.7.4</californium.version>
+    <bouncycastle.version>1.75</bouncycastle.version>
```

```suggestion
    <bouncycastle.version>1.77</bouncycastle.version>
```

could we switch to 1.77 - or is there a reason to stick with 1.75 for now?

---

Yes, we can switch.  1.77 came out after my last push I believe.